algorithms/symmetry/cosym/target.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         cb_op_to_primitive = data.change_of_basis_op_to_primitive_setting()
         data = data.change_basis(cb_op_to_primitive).map_to_asu()
 
-        order = lattice_ids.argsort()
+        order = lattice_ids.argsort().astype(np.uint64) # Convert to uint64 avoids crashes on Windows
```

Comment:
Could you expand this comment here? The commit message comment was clearer, but I had to read the commit message to better understand the code change.